guidelines/sc/22/pointer-target-spacing.html

```diff
@@ -10,6 +10,7 @@ <h4>Pointer Target Spacing</h4>
       <li><strong>Enlarge:</strong> A <a>mechanism</a> is available to change the CSS pixel size of each target, or its spacing, so there is an area with a width and height of at least 44 CSS pixels that includes it, and no other targets;</li>
         <li><strong>Inline:</strong> The target is in a sentence or block of text;</li>
         <li><strong>User agent:</strong> The size of the target is controlled by the <a>user agent</a> and is not modified by the author;</li>
+        <li><strong>Equivalent function:</strong> The target’s function is achievable through another component that has a target area of at least 24 by 24 CSS pixels;</li>
```

Comment:
Do we not need the same exceptions for the equivalent targets that are embodied in exception "User agent"? I.e., a dropdown navigation with item heights below 24px would be OK if there is a select with the same entries, but these would fail the 24x24px prescription (only pass via the user agent exception). Not sure whether this needs to be spelled out though...

Reply:
Crap, just realised I did the PR against master instead of the scha14-pointer-updates branch. Sorry, I'll update.